utFileMonitor.py

```diff
@@ -0,0 +1,119 @@
+import sys
+import os
+import re
+import time
+import threading
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../../../")
+sys.path.append(dir_path)
+
+from framework.core.logModule import logModule
+from framework.core.commandModules.consoleInterface import consoleInterface
+from interactiveShell import InteractiveShell
+
+class utFileMonitor:
+    """This Class monitors the log files. If there are any new lines added to the file, then reports back
```

Comment:
highlight that the class will monitor a file on the device connected to the session.